ir/type.cpp

```diff
@@ -223,6 +229,8 @@ void VectorType::fixup(const Model &m) {
   // TODO
 }
 
+expr VectorType::enforceIntOrVectorType() const { return true; }
```

Comment:
Please stick to the project's style :)

Reply:
Ah, you mean the formatting, not the naming.
This is what clang-format gave me by default.
Should the .clang-format be customized then?

Reply:
Okay, `.clang-tidy` with
```
---
Language:        Cpp
BasedOnStyle:  LLVM
AllowShortFunctionsOnASingleLine: Empty
...
```
did it.